client/setup.py

```diff
@@ -47,7 +47,7 @@ def _format_version() -> str:
     author_email="developer@starwhale.ai",
     version=_format_version(),
     description="An MLOps Platform for Model Evaluation",
-    long_description=open("../README.md", encoding="utf-8").read(),
+    long_description=open("./README.md", encoding="utf-8").read(),
```

Comment:
the wrong position, setup.py needs the readme of the parent dir.  the long description is for the pypi. https://pypi.org/project/starwhale/